internal/nmc/helper.go

```diff
@@ -74,3 +74,45 @@ func (h *helper) GetModuleEntry(nmc *kmmv1beta1.NodeModulesConfig, modNamespace,
 	}
 	return nil, 0
 }
+
+func findModuleStatus(statuses []kmmv1beta1.NodeModuleStatus, moduleNamespace, moduleName string) *kmmv1beta1.NodeModuleStatus {
```

Comment:
it looks a little bit strange that Spec has methods and Status has functions. Can we make them all methods and then decide upon in v2.1?

Reply:
Or we make them all functions? 🙂